src/components/alert/alert.tsx

```diff
@@ -49,15 +49,15 @@ export const IonAlert = ({
 
   return (
     <AlertStyled
-      data-testid="ion-alert"
+      data-testid='ion-alert'
       type={type}
       hideBackground={hideBackground}
     >
       <IonIcon type={icon} size={sizeIcon}></IonIcon>
```

Comment:
```suggestion
      <IonIcon type={icon} size={iconSize}></IonIcon>
```
maybe change it to iconSize?